detox/android/detox/src/full/java/com/wix/detox/LaunchArgs.java

```diff
@@ -29,6 +31,14 @@ String getUrlOverride() {
         return InstrumentationRegistry.getArguments().getString(DETOX_URL_OVERRIDE_ARG);
     }
 
+    String getDetoxServerUrl() {
```

Comment:
Hehe, makes a lot of sense :) Single responsibility FTW.

Reply:
Yes yes, very old debts